src/Microsoft.DotNet.Arcade.Sdk/tools/DefaultVersions.props

```
@@ -55,6 +55,12 @@
       Repo can set this property to true if it needs to use a different version of the compiler than the one in the dotnet SDK.
     -->
     <UsingToolMicrosoftNetCompilers Condition="'$(UsingToolMicrosoftNetCompilers)' == ''">false</UsingToolMicrosoftNetCompilers>
+
+    <!--
+      Use linkers from the Microsoft.NET.ILLink.Tasks package.
+      Repo can set this property to true if it needs to use a different version of the linker than the one in the dotnet SDK.
+    -->
+    <UsingToolMicrosoftNetLinkers Condition="'$(UsingToolMicrosoftNetLinkers)' == ''">false</UsingToolMicrosoftNetLinkers>
```

I'd prefer if we match the property name with the package name like we do for `UsingToolMicrosoftNetCompilers`:

```suggestion
      Use IL linker from the Microsoft.NET.ILLink.Tasks package.
      Repo can set this property to true if it needs to use a different version of the IL linker than the one in the dotnet SDK.
    -->
    <UsingToolMicrosoftNetILLinkTasks Condition="'$(UsingToolMicrosoftNetILLinkTasks)' == ''">false</UsingToolMicrosoftNetILLinkTasks>
```